rise/lib/services/questions.ts

```diff
@@ -12,3 +13,36 @@ export const getQuestions: QueryResolvers['questions'] = async () => {
   `
   )
 }
+
+export async function latestAnswerByUser(id: string): Promise<any[]> {
```

Comment:
See the `getQuestions` above for an approach which is easier to get typed lets us get rid of the `any`!